content/blog/2023/03/03/2023-03-09-jenkins-awards-2023-candidates.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Jenkins Contributor Awards - Voting Open"
+tags:
+- jenkins
+- community
+- contribute
+- awards
+- cdcon
+authors:
+- kmartens27
+opengraph:
+image: image:/images/post-images/2023/02/23/2023-02-23-cdf-awards/image1.png
```

Comment:
Needs indentation to indicate this is the opengraph image and does not need the second copy of `image`
```suggestion
  image: /images/post-images/2023/02/23/2023-02-23-cdf-awards/image1.png
```